api/src/test/java/org/openmrs/api/impl/ConditionServiceImplTest.java

```diff
@@ -109,6 +109,17 @@ public void getActiveConditions_shouldGetActiveConditions() {
 		Assert.assertEquals("2cc6880e-2c46-11e4-9138-a6c5e4d20fb7",activeConditions.get(0).getUuid());
 		Assert.assertEquals("2cc6880e-2c46-15e4-9038-a6c5e4d22fb7", activeConditions.get(1).getUuid());
 	}
+
+    /**
+	 * @see ConditionService#getAllConditions(Patient)
+	 */
+	@Test
+	public void getAllConditions_shouldGetAllConditions() {
+		List<Condition> conditions = conditionService.getAllConditions(patientService.getPatient(2));
+		Assert.assertTrue(conditions.size() == 2);
+		Assert.assertEquals("2cc6880e-2c46-11e4-9138-a6c5e4d20fb7",conditions.get(0).getUuid());
```

Comment:
Does the test include inactive conditions?

Reply:
@dkayiwa  I updated the PR.